OutSystems/outsystems-ui
# ROU-3789: Fix issue on layout native

This PR is for fixing an issue on layout native when the menu behavior is expandable.

### What was happening
- The CSS rules are incorrectly applied:
![image](https://user-images.githubusercontent.com/25321845/232450735-26a23ba8-a060-48ad-971b-937356194747.png)
![image](https://user-images.githubusercontent.com/25321845/232450828-85dd43c1-443f-450f-b4d1-fad5a7df88c3.png)

### What was done
- Removed the CSS that are causing the issue:
![image](https://user-images.githubusercontent.com/25321845/232450965-6e125a89-627e-4596-a55d-5d2097cc8841.png)
![image](https://user-images.githubusercontent.com/25321845/232451056-e0de135e-ba08-4ae8-8c7f-9e7f2fcde4b1.png)

### Test Steps
1. Open Sample Page
2. Check if the layout are correct
3. Enable the RTL
4. Expected: The layout will change the view based on RTL and the menu icon are visible

### Checklist
